src/standard/utils.html

```diff
@@ -367,50 +367,52 @@
      * @return {HTMLLinkElement} The link element for the URL to be loaded.
      */
     importHref: function(href, onload, onerror, optAsync) {
-      var link = document.createElement('link');
-      link.rel = 'import';
-      link.href = href;
-      var list = Polymer.Base.importHref.imported = 
-        Polymer.Base.importHref.imported || {};
-      var cached = list[link.href];
-      var imprt = cached || link;
+      var link = document.head.querySelector('[href="' + href + '"]');
       var self = this;
-      var loadListener = function(e) {
-        e.target.__firedLoad = true;
-        e.target.removeEventListener('load', loadListener);
-        e.target.removeEventListener('error', errorListener);
-        return onload.call(self, e);
-      };
-      var errorListener = function(e) {
-        e.target.__firedError = true;
-        e.target.removeEventListener('load', loadListener);
-        e.target.removeEventListener('error', errorListener);
-        return onerror.call(self, e);
-      };
-      if (onload) {
-        imprt.addEventListener('load', loadListener);
+
+      if (link == null) {
+        link = document.createElement('link');
+        link.rel = 'import';
+        link.__dynamicImport = true;
       }
-      if (onerror) {
-        imprt.addEventListener('error', errorListener);
+
+      if (optAsync) {
+        link.setAttribute('async', '');
```

Comment:
This conditional branch exists on its own to satisfy the constraints of a test. The test asserts that a link is imported async by checking for an `async` attribute on a link. If a link is imported synchronously at first, and then later imported asynchronously (as is the case in the test), then we must always set the `async` attribute even if we are relying on a "cached" (that is, already-done-loading) `link` tag for state.